src/agent/protocols/protos/agent.proto

```diff
@@ -1,6 +1,6 @@
 //
 // Copyright 2017 HyperHQ Inc.
-// Copyright 2019 Ant Financial
+// Copyright (c) 2019 Ant Financial
```

Comment:
While at it, please rename it to `Ant Group`?

Reply:
@bergwolf I have changed the copyright to `Copyright (c) 2019-2020 Ant Group`, thx.